eagleproject/core/blockchain.py

```diff
@@ -543,13 +543,16 @@ def maybe_store_stake_withdrawn_record(log, block):
     if len(db_staked) > 0:
         return db_staked[0]
 
+    is_staked_event = log["topics"][0] == SIG_EVENT_STAKED
+
     staked = OgnStaked(
         tx_hash=tx_hash,
         log_index=log_index,
         block_time=block.block_time,
         user_address="0x" + log["topics"][1][-40:],
-        is_staked=log["topics"][0] == SIG_EVENT_STAKED,
+        is_staked=is_staked_event,
         amount=int(_slot(log["data"], 0), 16) / 1e18,
+        staked_amount=int(_slot(log["data"], 1), 16) / 1e18 if not is_staked_event else 0,
```

Comment:
should we also store the staked_duration? I think that will be helpful for the business to run some analysis on user's staking behavior.

Reply:
Will add it